indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -77,7 +77,10 @@ def _process(self):
         form = SessionForm(obj=FormDefaults(colors=self._get_random_color()))
         if form.validate_on_submit():
             create_session(self.event_new, form.data)
-            return jsonify_data(html=_render_session_list(self.event_new))
+            sessions = [{'id': s.id, 'title': s.title, 'colors': s.colors}
+                        for s in self.event_new.sessions if not s.is_deleted]
+            return jsonify_data(html=_render_session_list(self.event_new),
+                                sessions=sessions)
```

Comment:
I don't think this wrap is needed (but I think it's more readable if you put `sessions=session, html=...` instead of the other way round)